test/abi.decodeParameter.js

```diff
@@ -635,6 +635,9 @@ describe('lib/solidity/coder', function () {
                         '0000000000000000000000000000000000000000000000000000000000000060' +
                         '0000000000000000000000000000000000000000000000000000000000000006' +
                         '737472696e670000000000000000000000000000000000000000000000000000'})
+
+        test( { types: ['string'], expected: ["0x"],
+                values: '0x000000000000000000000000000000000000000000000000000000000000002000000000000000000000000000000000000000000000000000000000000000023078000000000000000000000000000000000000000000000000000000000000'})
```

Comment:
The change in code seems ok, but this test should decode '0x' in to '0x' string instead of `'0x000000000000000000000000000000000000000000000000000000000000002000000000000000000000000000000000000000000000000000000000000000023078000000000000000000000000000000000000000000000000000000000000' `?

Reply:
Yes this test is exactly doing what is required.  Its binary value return from contract
> 0x000000000000000000000000000000000000000000000000000000000000002000000000000000000000000000000000000000000000000000000000000000023078000000000000000000000000000000000000000000000000000000000000

to `"0x"` string.